server/src/main/java/org/elasticsearch/cluster/metadata/MetadataIndexStateService.java

```diff
@@ -925,7 +940,7 @@ public ClusterState execute(final ClusterState currentState) {
                     return allocationService.reroute(updatedState, "indices opened [" + indicesAsString + "]");
                 }
             },
-            ClusterStateTaskExecutor.unbatched()
```

Comment:
Not sure we want to batch this in with the other operations? It seems like this is not something that ILM will do in a hot loop right?
I could see some strange side effects of batching open and close index updates into a single batch in some edge cases.